core/src/main/java/com/novoda/noplayer/internal/exoplayer/SecurityDowngradingCodecSelector.java

```diff
@@ -20,23 +22,22 @@ public static SecurityDowngradingCodecSelector newInstance() {
     }
 
     @Override
-    public MediaCodecInfo getDecoderInfo(String mimeType, boolean contentRequiresSecureDecoder)
-            throws MediaCodecUtil.DecoderQueryException {
-        return internalMediaCodecUtil.getDecoderInfo(mimeType, USE_INSECURE_DECODER);
+    public List<MediaCodecInfo> getDecoderInfos(String mimeType, boolean requiresSecureDecoder) throws MediaCodecUtil.DecoderQueryException {
+        return internalMediaCodecUtil.getDecoderInfos(mimeType, USE_INSECURE_DECODER);
     }
 
     @Override
-    public MediaCodecInfo getPassthroughDecoderInfo() {
+    public MediaCodecInfo getPassthroughDecoderInfo() throws MediaCodecUtil.DecoderQueryException {
```

Comment:
How come this was fine before? Is it a `RuntimeException`? Should it be checked?